components/dashboard/sections/transactions/filters.tsx

```diff
@@ -185,16 +221,19 @@ export const filters: FilterComponentConfigs<FilterValues, FilterMeta> = {
       );
     },
   },
-  isRefund: {
-    labelMsg: defineMessage({ defaultMessage: 'Is Refund', id: 'o+jEZR' }),
-    Component: ({ intl, ...props }) => {
-      const options = React.useMemo(() => {
-        return [true, false].map(value => ({
-          label: i18nIsRefund(intl, value),
-          value,
-        }));
-      }, [intl]);
-      return <ComboSelectFilter options={options} {...props} />;
+  merchantId: {
```

Comment:
We're missing something like this for "group".

Reply:
I've intentionally left out the input component for `group`, for that we need something that handles multiple IDs/strings. But I'm also thinking that the buttons to "View related transactions" that adds this filter might be enough?

Reply:
Have added a Group ID filter component now, using the ComboSelectFilter (modified to work also for this usecase of a creatable multi-select without searching)